sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -472,7 +482,7 @@ export abstract class LinkEntity<LinkT extends Receiver | AwaitableSender | Requ
    * Ensures that the token is renewed within the predefined renewal margin.
    * @returns {void}
    */
-  private _ensureTokenRenewal(): void {
+  private _ensureTokenRenewal(operationOptions?: OperationOptionsBase): void {
```

Comment:
Can we make this argument non-optional? It could still accept undefined if that's needed:

```typescript
private _ensureTokenRenewal(operationOptions: OperationOptionsBase | undefined);
```

Reply:
Yes.

Reply:
+1 on this.